<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color :'white'</string>
   </property>
   <widget class="QLabel" name="AppName">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color : rgb(118, 173, 232);
border-radius :10px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Broadcast&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;With Attachment &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>161</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color : rgb(177, 220, 242);
border-radius : 15px;
}</string>
    </property>
    <property name="text">
     <string>Type Message :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="MessageBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>441</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
background-color: rgb(118, 173, 232);
border-radius : 15px;
border-style: none;

}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunBroadcast">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color : rgb(16, 166, 235);
border-radius :25px;
}</string>
    </property>
    <property name="text">
     <string>Run Broadcast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>910</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color : rgb(177, 220, 242);
border-radius :25px;
}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
